bzl/tools/user.bazelrc

```diff
@@ -0,0 +1,24 @@
+## Override the WORKSPACE definitions of repos:
+build --override_repository=xbyak=/path/to/xbyak
```

Comment:
Shouldn't this `/path/to/xbyak` placeholder be filled in here?

Reply:
Sorry,I'll add some documentation.  This file is a template; to use it you would copy it to the root dir and replace /path/to/xbyak with the actual path.  The root .bazelrc file contains a line that loads user.bazelrc if it exists.

Based on my experience, I think using overrides in user.bazelrc is probably a better way of using embedded repo code, rather than using `local_repository` in the WORKSPACE. At least I'm leaning in that direction; both ways work.